device.c

```diff
@@ -133,6 +149,16 @@ static int vcam_try_fmt_vid_cap(struct file *file,
         f->fmt.pix.height = dev->output_format.height;
     }
 
+    if (dev->conv_res_on) {
+        const struct v4l2_frmsize_discrete *sz =
+                v4l2_find_nearest_size(vcam_sizes,
+                                       VCAM_SIZES, width,
```

Comment:
Ditto. Replace `VCAM_SIZES` with `ARRAY_SIZE(vcam_sizes)` since it is a compile-time constant.